README.md

```diff
@@ -1 +1,38 @@
 # DataStructures
+
+<h4><u>Dynamic array:</u></h4>
+<p>An array that changes depending on input quantities. 
+These are best used for storing data with unknown limits.
+The opportunity cost of dynamic arrays can be over allocation 
+of memory, or the time it takes to iterate through the list (severity of
+this depends on list architecture.</p>
+
+<h4><u>Stack:</u></h4>
+<p>Stacks, aka Last-in-First-out (LIFO), allow you to control
+your data in a different way than a list. 
```

Comment:
Move your closing print tag (</p>) to this line. You should not nest ordered lists inside of print tags. HTML is very forgiving about these kind of things, but it is not good practice.